data/ColorScheme.json

```diff
@@ -58,7 +58,8 @@
         "textColor": [255, 255, 255, 255]
     },
     "Amaterasu": {
-        "backgroundBox": [0, 0, 0, 80],
+        "backgroundBoxColor": [0, 0, 0, 80],
```

Comment:
perhaps keep the old color box name apart from the size which we can perhaps find a better way of doing